packages/core/package.json

```diff
@@ -38,6 +38,7 @@
         "lint:scss": "sass-lint",
         "lint:es": "es-lint",
         "lint-fix": "es-lint --fix && sass-lint --fix",
+        "prepack": "run-s dist",
```

Comment:
The `dist` command also depends on `compile`. Should we include that here in the prepack command?